test/integration/utils_test.go

```diff
@@ -60,14 +60,14 @@ var (
 	// httpBinImage is the container image name we use for deploying the "httpbin" HTTP testing tool.
 	// if you need a simple HTTP server for tests you're writing, use this and check the documentation.
 	// See: https://github.com/postmanlabs/httpbin
-	httpBinImage = "kennethreitz/httpbin"
+	httpBinImage = "kong/httpbin:0.1.0"
```

Comment:
Can we put this into the `test` package (one level higher) so that it can be shared between e2e and integration tests? I'm OK in either doing this in this or a subsequent PR

And then the same can be done with echo image `const` as well?